frontend/language/src/nb.json

```
@@ -1683,6 +1683,14 @@
   "ux_editor.component_title.TextArea": "Langt svar",
   "ux_editor.component_unknown": "Ukjent komponent",
   "ux_editor.conditional_rendering_connection_header": "Betingede renderingstilkoblinger",
+  "ux_editor.config.warning_duplicates.heading": "Du har den samme ID-en på flere komponenter",
+  "ux_editor.config.warning_duplicates.heading2": "For å fikse problemet, må du gjøre dette:",
+  "ux_editor.config.warning_duplicates.list1": "Gå til Gitea for å endre filen med feil.",
```

Maybe we can rename these too to make it more clear? 😄 
Here is an example for the first one
```suggestion
  "ux_editor.config.warning_duplicates.solution_gitea": "Gå til Gitea for å endre filen med feil.",
```